src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -26,9 +27,12 @@ const getClassNames = classNamesFunction<
 const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsListProps> = ({
     twins,
     searchedProperties,
-    styles
+    onTwinSelection,
+    styles,
+    adapter
```

Comment:
nit: let's keep these in alphabetical order please.

Reply:
is it possible to have a linting rule that formats this on save somehow?